terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,75 @@
+# Define all hardcoded local variable and local variables looked up from data resources 
+locals {
+  stack_name                 = "filing-create" # this must match the stack name the service deploys into
+  name_prefix                = "${local.stack_name}-${var.environment}"
+  global_prefix              = "global-${var.environment}"
+  service_name               = "overseas-entities-api"
+  container_port             = "8080"
+  eric_port                  = "10000"
+  docker_repo                = "overseas-entities-api"
+  kms_alias                  = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority  = 64
+  lb_listener_paths          = ["/transactions/*/overseas-entity*", "/private/transactions/*/overseas-entity/*"]
```

Comment:
We also want to add the healthcheck path to the listener paths

Reply:
done